packages/lib/modules/portfolio/PortfolioClaim/ClaimProtocolRevenueModal.tsx

```diff
@@ -51,11 +51,12 @@ export default function ClaimProtocolRevenueModal({ isOpen, onClose }: Props) {
 
   const claimTxHash = transactionSteps.lastTransaction?.result?.data?.transactionHash
 
-  const rewards: HumanTokenAmountWithAddress[] = rewardsDataSnapshot
+  const rewards: HumanTokenAmount[] = rewardsDataSnapshot
     .filter(reward => !bn(reward.balance).isZero())
     .map(reward => ({
       tokenAddress: reward.tokenAddress as Address,
       humanAmount: reward.humanBalance,
+      symbol: '-',
```

Comment:
Again, why do this? Will it not cause problems or bad UX?

Reply:
symbol was not needed here. I added it during an alternative refactor and forgot about removing it later.